packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -69,6 +69,8 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
   private inverted = false;
 
   // Viewport Properties
+  private colormap: ColormapPublic;
+  private voiRange: VOIRange;
```

Comment:
These are a little bit odd, same for inverted above. Since the volume viewport can have two volumeIds, which of them we are storing its colormap and voiRange or inverted? Maybe these should be a Map()